ieeg_motorMiller2007/dataset_description.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Name": "Miller_et_al_2007_Jneurosci",
+    "BIDSVersion": "1.0.2",
+    "License": "",
```

Comment:
Since this is an example not a template better not include empty fields. We wouldn't like people to mimic this.

Reply:
BTW there are more empty values in other JSON files.